src/containers/home/index.tsx

```diff
@@ -72,15 +74,19 @@ const ViewMoreButton = styled(LinkButton)`
   margin: 1em;
 `;
 
-const ANNOUNCEMENTS_LIMIT = 3;
+const CARD_ROW_LIMIT = 3;
 
-export type HomeContainerProps = AnnouncementsDataProps;
+export type HomeContainerProps = {
```

Comment:
type vs interface? why type?

Reply:
```
type name = {
 ...
}
```
is the same as
```
interface name {
 ...
}
```

Reply:
We should definitely do interface for consistency though 👍